### PR TITLE
Rebalance the ridiculous cargo prices, add new items to the list

### PR DESCRIPTION
**Pull request by XElectricX**
## About The Pull Request

Lowered the price of practically everything or increased the value by adding more to the orders. Things of significant addition are purchasable spec armors: B18 for 200 points, B17 and Pyro for 100. All gun orders come with some ammo now. Resorted a lot of of the list and removed useless orders like SMES coils and the science category. Added an imports category: features several gimmick weapons for sale at 30 points, with the holdout pistol being slightly lower. Each comes with 1 to 3 magazines/clips/boxes of ammo, just enough for a few fights if played conservatively. I've only added non-ERT weapons to keep balance intact. SWAT masks can be ordered, 4 masks for 40 points. Protect from one hugger. A riot armor set was added for sale. This particular "marine" variant has slightly lower melee armor but gains equal bullet armor. The riot helmet does not stop huggers.

Ammo boxes were made for each of the new guns, except for the revolver and auto shotgun. The boxes use the same sprites the other weapons used. They don't need them anymore. Each box costs 20 points and comes with 10 magazines.

## Why It's Good For The Game

The cargo section has glaringly over priced items which makes anything outside turrets, spec guns, or materials not worth buying. This will make prices more fair. Another purpose for this PR to create better incentives for the marines and crew to gain more points. The new armors, weapons, and ammo benefit the marines more for collecting points. Looking for feedback in case anything is too cheap/expensive.

## Changelog
:cl:
add: Orderable bulk ammo boxes.
add: Many, many new shiny items added to cargo orders.
del: Removed a lot of junk cargo orders.
balance: Slashed a lot of cargo prices across the board. More fair prices for orders.
/:cl:

### CODE REVIEW

**Comment by Timmeye** [comment]
So are we gonna get money for buying oil barrels now?

**Comment by XElectricX** [comment]
No, no, you SELL the ~~oil barrels~~ mining crates if that ever gets merged to afford these.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -2504,7 +2181,361 @@ SUPPLIES
...
+					)
+	cost = 30
+	containertype = /obj/structure/closet/crate/weapon
+	containername = "\improper AK-47U Battle Carbine Crate"
+	group = "Imports"
+
+/datum/supply_packs/imports_m16	//Vietnam time
+	name = "FN M16A Assault Rifle"
+	contains = list(
+					/obj/item/weapon/gun/rifle/m16
```
Comment (spookydonut) [inline code/datums/supplypacks.dm:2416]:
```suggestion
					/obj/item/weapon/gun/rifle/m16,
```

Comment (XElectricX) [reply code/datums/supplypacks.dm:2416]: Whoops, thank you for pointing it out.

**Comment by psykzz** [comment]
Why was the type 71 added, isn't that a UPP weapon?

**Pull request merged**